radio/src/gui/480x272/model_setup.cpp

```diff
@@ -636,8 +647,11 @@ bool menuModelSetup(event_t event)
            IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),   // Receiver 1
            IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),   // Receiver 2
            IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),   // Receiver 3
-
-         TRAINER_ROWS
+//OW
+//         TRAINER_ROWS
```

Comment:
don't hesitate to remove the line completely, if you put it in comments it only raises questions

Reply:
thx. Done.